capstone-app/src/app/modules/maps/map/map.component.ts

```diff
@@ -89,12 +89,61 @@ export class MapComponent implements OnInit{
     this.activeMarkerOutput.emit(this.activeMark);
   }
 
+  getCurrentLocation () {
+    let filterResult: string;
+    let prevResult: string = 'blank';
+    navigator.geolocation.getCurrentPosition(position => {
+      this.position = {
+        lat: position.coords.latitude,
+        lng: position.coords.longitude,
+      };
+      this.location = new google.maps.LatLng(this.position);
+      this.geocoder.geocode({'location': this.position}, (results, status) => {
+        if (status === 'OK') {
+          for (let result of results) {
+            filterResult = result.formatted_address;
+            if (filterResult.match(/[1-1000]/)) {
+              continue;
+            }
+            if (filterResult.includes('County')){
+              continue;
+            }
+            if (!filterResult.includes(',')) {
```

Comment:
No commas at all? Won't that filter out city, state, country?

Reply:
so one of the that just filter out the option that is just Country because one of the options is just USA instead of Miami, Fl, USA